### ISSUE TITLE
Function arguments can get incorrectly appended to comments

### ISSUE THREAD

**Issue by alastair-harrison** [cheshirekow/cmake_format#47]
```bash
$ clang-format --version
0.3.6
```

When I run `cmake-format` on a code block like this:
```cmake
my_function(
  item1

  # Some comment
  item2
)

```
`cmake-format` gives me the following output:
```cmake
my_function(item1 # Some comment item2)
```

There should be a line break before `item2`.

**Comment by sum01** [comment]
Isn't this more or less the same as https://github.com/cheshirekow/cmake_format/issues/46 or at least very related? It seems that it's probably the cause of this issue anyways.

**Comment by bergercookie** [comment]
> Isn't this more or less the same as #46 or at least very related? It seems that it's probably the cause of this issue anyways.

Adding to this, consider the following example


```
if(NOT CMAKE_BUILD_TYPE)
  set(CMAKE_BUILD_TYPE Release CACHE STRING "Choose the type of build." FORCE)
  # a comment here
endif()
```

I assume in this case, cmake-format (incorrectly) tries to join the string to the previous line, but this also results in the following exception:

`ValueError: invalid width -1 (must be > 0) `

Here's the full traceback:

![image](https://user-images.githubusercontent.com/5816719/41768030-14b7966c-7603-11e8-9a84-e5ae88c95eb6.png)

**Comment by cheshirekow** [comment]
Fixed I believe in `v0.4.0`. If you're able, please try https://github.com/cheshirekow/cmake_format/releases/tag/v0.4.0rc1 and lte me know how it goes

**Comment by alastair-harrison** [comment]
Yes, seems to maintain the semantics of the function call now. Thanks.
